queries.sql

```diff
@@ -0,0 +1,127 @@
+-- Please note that SQLite syntax will be in use.
+
+.mode column
+.header on
+
+PRAGMA foreign_keys = OFF;
+
+DROP TABLE IF EXISTS organization;
+DROP TABLE IF EXISTS channel;
+DROP TABLE IF EXISTS user;
+DROP TABLE IF EXISTS message;
+DROP TABLE IF EXISTS user_channel;
+
+PRAGMA foreign_keys = ON;
+
+CREATE TABLE organization (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name TEXT NOT NULL
+);
+
+CREATE TABLE channel (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  organization_id INTEGER REFERENCES organization(id) ON DELETE CASCADE NOT NULL,
+  name TEXT NOT NULL
+);
+
+CREATE TABLE user (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name TEXT NOT NULL
+);
+
+CREATE TABLE message (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  user_id INTEGER REFERENCES user(id) ON DELETE CASCADE NOT NULL,
+  channel_id INTEGER REFERENCES channel(id) ON DELETE CASCADE NOT NULL,
+  content TEXT NOT NULL,
+  post_time TEXT NOT NULL
+);
+
+CREATE TABLE user_channel (
+  user_id INTEGER REFERENCES user(id),
+  channel_id INTEGER REFERENCES channel(id)
+);
+
+INSERT INTO organization (name) VALUES ("Lambda School");
+
+INSERT INTO user (name) VALUES ("Alice");
+INSERT INTO user (name) VALUES ("Bob");
+INSERT INTO user (name) VALUES ("Chris");
```

Comment:
Can also use`insert` in one line like this `INSERT INTO user ( name ) VALUES ("Alice"), ("Bob"), ("Chris")`